src/filesystem/impls/filer/lib/CSSRewriter.js

```diff
@@ -28,6 +28,7 @@ define(function (require, exports, module) {
         function aggregate(content, callback) {
             var urls = [];
             var urlRegex = new RegExp('url\\([\\\'\\"]?([^\\\'\\"\\)]+)[\\\'\\"]?\\)', 'g');
+            var urlQuotesRegex = new RegExp(/[\"\'](.*?\.css)[\"\']/, 'g');
```

Comment:
Why not just adapt the portion of the regex above that deals with the part inside the `url(...)`?  That would be `/[\\\'\\"]([^\\\'\\"\\)]+)[\\\'\\"]/`?

cc @Pomax